projects/inception-lrt/index.js

```diff
@@ -7,14 +7,44 @@ async function tvl_eth(api) {
   const strategies = await api.multiCall({  abi: 'address:strategy', calls: vaults})
   const tokens = await api.multiCall({  abi: 'address:underlyingToken', calls: strategies})
   // const result = total_deposited.map((deposited, index) => deposited - total_withdraw[index]);
-  if (api.timestamp > 1715181240) {
-    const staked = await api.call({ abi: 'uint256:totalAssets', target: '0xf073bAC22DAb7FaF4a3Dd6c6189a70D54110525C' })
-    api.add(ADDRESSES.null, staked)
-  }
   api.add(tokens, total_deposited)
 }
 
+async function tvl_linea(api) {
+    const staked = await api.call({ abi: 'uint256:totalSupply', target: '0xd08C3F25862077056cb1b710937576Af899a4959' })
+    api.add(ADDRESSES.null, staked)
+}
+
+async function tvl_mode(api) {
```

Comment:
these are bridged tokens right? users dont interact with inception or genesis contracts on these chains to deposit/withdraw right?

Reply:
yes sir

Reply:
hi @Sk1mer12 was waiting for you to remove totalSupply/tvl on other chains, sorry for not being clear